scopesim/effects/metis_lms_trace_list.py

```diff
@@ -580,6 +584,13 @@ def __init__(self, **kwargs):
 
         lam, efficiency = self.make_ter_curve(wcal, wavelen)
 
+        # HACK: Somehow we end up with duplicate keywords here. This hack
+        #       should not be necessary at all! Investigate what's really
+        #       going on...
```

Comment:
Maybe because you removed the `copy()` in `__init__()`? You directly set `self.meta = self._class_params`, which means that all `MetisLMSEfficiency` instances share the same `.meta`.

Just guessing. And since I'm already started with 'single comments', I'll keep doing that :-).